pom.xml

```diff
@@ -77,8 +77,8 @@
 
   <properties>
     <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
-    <maven-surefire-plugin.version>3.0.0-M4</maven-surefire-plugin.version>
-    <maven-failsafe-plugin.version>3.0.0-M4</maven-failsafe-plugin.version>
+    <maven-surefire-plugin.version>3.0.0-M6</maven-surefire-plugin.version>
```

Comment:
just released 3.0.0-M7, can we update?
https://lists.apache.org/thread/zgqj6bntd7m3x0wpcw0zopy3z5rjbr9r

Reply:
of course